### PR TITLE
feat(login): added password toggle + tests

### LINKED ISSUE #84: Feat: Add show/hide password toggle functionality on password fields
**Is your feature request related to a problem? Please describe.**
At the moment password field has no show/hide visibility option on the login and register form.

**Describe the solution you'd like**
Password fields need to have a show/hide visibility option so they can check their input on the field.

**Describe alternatives you've considered**
Open to contributor's idea as long as it fits the application well.

**Additional context**
NA

### PR DESCRIPTION
**Pull request by ruppysuppy**
### Description

Added Password Toggle feature (Login Form)

Fixes #84 

### Type of Change:

- User Interface

### How Has This Been Tested?
Added automated tests

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] I have written Kotlin Docs whenever is applicable

### CODE REVIEW

File: src/login/Login.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default function Login() {
     const [errorMessage, setErrorMessage] = useState(null);
     const {isAuth,login} = useContext(AuthContext);
     const [user, setUser] = useState(null);
+    const [isPasswordShown, setIsPasswordShown] = useState(false)
     
     const handleSubmit = async e => {
         e.preventDefault();
```
Comment (mtreacy002) [inline src/login/Login.jsx:15]: @ruppysuppy . I've tested the change you've made. The toggle checkbox has unintentionally got included inside the payload body and causing an error response `Wrong username and password` even though it was using the correct combo.
Here's the screenshot UI response:
Screen Shot 2020-09-10 at 10 13 14 am

Here're my findings from debugging:
* incorrect payload body caused by code change
Screen Shot 2020-09-10 at 10 15 08 am

* the expected payload as per tested on deevelop branch
Screen Shot 2020-09-10 at 10 17 16 am

You can fix this by filtering out the checkbox from being included in the payload.

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@ruppysuppy , the code change carries a bug. Please see my comment below.

**Review by mtreacy002 — APPROVED** [review]
Sorry, @ruppysuppy. I made a mistake. Your code change works. I made the mistake in thee username - password combo after changing username for testing other PR. Sorry for jumping to conclusion. Great job with the toggle box.

Successful test
ezgif com-video-to-gif (7)

**Comment by mtreacy002** [comment]
@anitab-org/bridgeintech-maintainers . Can someone else please give their review on this PR? I've tested and approved the code change. Thanks

**Review by isabelcosta — APPROVED** [review]
Seems to make sense :) Thank you for contributing @ruppysuppy !

**Pull request merged**